tests/Unit/Controllers/DriftControllerTest.php

```
@@ -46,6 +47,11 @@ protected function tearDown(): void
         parent::tearDown();
     }
 
+    protected function makePostRequest(array $body = [], string $uri = '/'): \StratFlow\Tests\Support\FakeRequest
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$uri'. (undefined)

(UnusedFormalParameter)